apps/client/src/pages/LocationPage.js

```diff
@@ -40,8 +40,9 @@ const LocationPage = ({ topic }) => {
         action: `postcode - ${slug.replace("-", " ")}`,
         name: address.postalCode.substring(0, 4),
       });
-      context.address = address;
-      history.push(geturl(routes.address, { slug }));
+
+      context.autofillData.address = address;
```

Comment:
We only store `address` in `autofillData`, why not keep it `address`? Is this for future features?

In the refresh branch this is changed to `context.address[slug]` because we're saving different address per slug. Don't know if this helps thinking about different naming. Let's discuss?